libs.json_simple/src/org/netbeans/libs/json_simple/Bundle.properties

```diff
@@ -1,44 +1,19 @@
-# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#Licensed to the Apache Software Foundation (ASF) under one
+#or more contributor license agreements.  See the NOTICE file
+#distributed with this work for additional information
+#regarding copyright ownership.  The ASF licenses this file
+#to you under the Apache License, Version 2.0 (the
+#"License"); you may not use this file except in compliance
+#with the License.  You may obtain a copy of the License at
```

Comment:
This seems unlike the license headers in other .properties files. No space before #, etc.

Reply:
I copied it from here. http://www.apache.org/legal/src-headers.html is this the incorrect place to get the header information?

Reply:
Let me know how you would like me to proceed